src/lib/y2partitioner/dialogs/partition_type.rb

```diff
@@ -44,6 +44,10 @@ def validate
           !value.nil?
         end
 
+        def init
+          self.value = items.first.first
```

Comment:
I expect that you use here `@ptemplate.type` which is in beginning set to reasonable default by caller